.github/workflows/stubtest_stubs.yml

```
@@ -1,4 +1,4 @@
-name: Test
+name: Run stubtest on stubs
```

```suggestion
name: Third-party stubtest
```

Could you also change this line to something like "Stdlib stubtest", so that it's easy to distinguish the two workflows:

https://github.com/python/typeshed/blob/f8ae34c34e01736ed7f38c35ed068420ee74fc71/.github/workflows/stubtest_stdlib.yml#L1

It would also be good to change the name of this workflow file so that it has "third-party" in it